### PR TITLE
9 adds GET /api/reviews/:review_id/comments functionality

### PR DESCRIPTION
**Pull request by Asileka**

### CODE REVIEW

File: __tests__/ncgamesapp.test.js [hunk 1]
```diff
@@ -205,3 +204,24 @@ describe("GET:/api/reviews", () => {
...
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("review_id");
+        });
+      });
+  });
+});
```
Comment (HeatherEver) [inline __tests__/ncgamesapp.test.js:252]: please can you add some error handling to this endpoint - 
- invalid review_id
- review not found
also what if the review has no comments - we would still like to return a 200 just with no comments to display

**Review by HeatherEver — COMMENTED** [review]
Nice work Kat - just think about the 200 test for a review with no comments, there made need to be some changes in your model or SQL query

File: __tests__/ncgamesapp.test.js [hunk 2]
```diff
@@ -205,3 +204,49 @@ describe("GET:/api/reviews", () => {
...
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("review id not found");
+      });
+  });
+  it("GET /api/reviews/1/comments  review_id with no comments throws 404 error", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("no comments found");
```
Comment (Angaca) [inline __tests__/ncgamesapp.test.js]: When we have a valid review with no comments on it we want to receive a 200 and an empty array. Imagine this being on a website would you expect an error if no one hasn't commented yet?

**Review by Angaca — CHANGES_REQUESTED** [review]
Please have a look at the comment I left :)

File: models/ncgamesmodel.js [hunk 3]
```diff
@@ -61,3 +60,41 @@ exports.fetchUsers = () => {
...
+    return connection
+      .query(
+        `SELECT *
+          FROM reviews 
+          WHERE review_id = $1 ;`,
+        valuesArr
+      )
+      .then((results) => {
+        console.log(results.rows.length + " results length in checkId models");
+        console.log(results.rows + " results.rows in checkId models");
```
Comment (Angaca) [inline models/ncgamesmodel.js]: remove logs

File: models/ncgamesmodel.js [hunk 4]
```diff
@@ -61,3 +60,41 @@ exports.fetchUsers = () => {
...
+        if (!results.rows.length) {
```
Comment (Angaca) [inline models/ncgamesmodel.js]: you can use the property rowCount instead of checking the rows.length

**Review by Angaca — APPROVED** [review]
Have a look at the comments for some very small changes but this is ready for merge!

**Pull request merged**